app/scripts/modules/ui/ODataMasterView.js

```diff
@@ -0,0 +1,145 @@
+const DataGrid = require('./datagrid/DataGrid.js');
+const UIUtils = require('./datagrid/UIUtils.js');
+const EntryLog = require('../utils/EntryLog.js');
+
+const COLUMNS = [{
+    id: "name",
+    title: "Name",
+    sortable: true,
+    align: undefined,
+    nonSelectable: false,
+    weight: 40,
+    visible: true,
+    allowInSortByEvenWhenHidden: false,
+    disclosure: true,
+    sortingFunction: function (a, b) {
+        return DataGrid.SortableDataGrid.StringComparator("name", a, b);
+    }
+},
+    {
+        id: "method",
+        title: "Method",
+        sortable: true,
+        align: undefined,
+        nonSelectable: false,
+        weight: 10,
+        visible: true,
+        allowInSortByEvenWhenHidden: false,
+        sortingFunction: function (a, b) {
+            return DataGrid.SortableDataGrid.StringComparator("method", a, b);
+        }
+    },
+    {
+        id: "status",
+        title: "Status",
+        sortable: true,
+        align: undefined,
+        nonSelectable: false,
+        weight: 10,
+        visible: true,
+        allowInSortByEvenWhenHidden: false,
+        sortingFunction: function (a, b) {
+            return DataGrid.SortableDataGrid.NumericComparator("status", a, b);
+        }
+    },
+    {
+        id: "note",
+        title: "Detail",
+        sortable: true,
+        align: undefined,
+        nonSelectable: false,
+        weight: 20,
+        visible: true,
+        allowInSortByEvenWhenHidden: false,
+        sortingFunction: function (a, b) {
+            return DataGrid.SortableDataGrid.StringComparator("note", a, b);
+        }
+    }];
+
+function ODataMasterView(domId, options) {
+
+    this.oContainerDOM = document.getElementById(domId);
+    this.oEntryLog = new EntryLog();
+
+    this.onSelectItem = function(oSelectedData) {};
+    this.onClearItems = function(oSelectedData) {};
+    if (options) {
+        this.onSelectItem = options.onSelectItem || this.onSelectItem;
+        this.onClearItems = options.onClearItems || this.onClearItems;
+    }
+
+    const oClearButton = this._createClearButton();
+    this.oContainerDOM.appendChild(oClearButton);
+
+    this.oDataGrid = this._createDataGrid();
+    this.oContainerDOM.appendChild(this.oDataGrid.element);
+
+    chrome.devtools.network.getHAR(async function (result) {
+        const entries = result.entries;
+        if (!entries.length) {
+            console.warn("No requests found by now");
+        }
+        entries.forEach(this.logEntry.bind(this));
+        chrome.devtools.network.onRequestFinished.addListener(this.logEntry.bind(this));
+    }.bind(this));
+
+}
+
+ODataMasterView.prototype.logEntry = function(oEntry) {
```

Comment:
minor: this function could be underscored as private. We already use this convention elsewhere in this code